pkg/infra/iac2/templates_compiler.go

```diff
@@ -329,7 +335,22 @@ func (tc TemplatesCompiler) resolveStructInput(resourceVal *reflect.Value, child
 				tmpl, err := tc.getNestedTemplate(path.Join(
 					camelToSnake(resourceVal.Type().Name()),
 					camelToSnake(correspondingStruct.Type().Name()),
-				))
+				), template.FuncMap{
+					// parseField parses a single field from the supplied value
+					"parseField": func(val reflect.Value, fieldName string) (string, error) {
```

Comment:
Unused?

Reply:
yeah i wasnt entirely sure where this was used